.gitlab/dev_container_deploy/docker_linux.yml

```diff
@@ -116,15 +116,17 @@ dev_master-a6:
     !reference [.on_main_a6]
   needs:
     - docker_build_agent6
+    - docker_build_agent6_arm64
     - docker_build_agent6_jmx
+    - docker_build_agent6_jmx_arm64
     - docker_build_agent6_py2py3_jmx
   variables:
     IMG_REGISTRIES: dev
   parallel:
     matrix:
-      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-amd64
+      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-arm64
         IMG_DESTINATIONS: agent-dev:master,agent-dev:master-py2
-      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-jmx-amd64
+      - IMG_SOURCES: ${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-jmx-amd64,${SRC_AGENT}:v${CI_PIPELINE_ID}-${CI_COMMIT_SHORT_SHA}-6-arm64
```

Comment:
Should this be `-6-jmx-arm64`? (I think you may be missing the `jmx` part)

Reply:
Not my best copy-paste, thanks ! Fixed in [8edd2e2](https://github.com/DataDog/datadog-agent/pull/11877/commits/8edd2e28038944bab42ec796c1302df61e5e8adc)